### DISCUSSION TITLE
Problems with Jamulus on Ipad

### DISCUSSION THREAD

**Discussion by Lightspacer** [jamulussoftware/jamulus#2671] [Bug Investigation]
Hi,
I am very glad that Jamulus now works on my Ipad. That is super! The sound via Ipad is ok, but when I connect to my B & O beolit 17 boxes via Bluetooth I get distorted sounds. 
Is it possible to solve this problem?
Jerry

**Comment by rdica** [comment]
I don't believe the iOS app currently available is an officially supported release from Jamulus. I suggest contacting the apps authors for support.

**Reply by ann0see** [reply]
No it is not. The next release will publish unsigned .ipa files but they're not to be considered official - similar to the Android version.

**Comment by Lightspacer** [comment]
How can I connect to the authors?

**Reply by rdica** [reply]
I imagine you'd look up the app in the app store?

**Reply by Lightspacer** [reply]
I tried to look in the app, to find out how I could send a message to the authors, but I cannot find it. 
Do they have an e-mail address? Do they check this forum once in a while?

**Reply by Lightspacer** [reply]
I see that Volker Fischer is the main author, but I cannot figure out how to send him a message…

**Reply by pljones** [reply]
Could you paste a screenshot of the "Help"->"About Jamulus..." page "About" tab, if possible?

**Reply by rdica** [reply]
Is this the thing you installed:
https://apps.apple.com/sn/app/jamulus2-0/id1609844773

**Reply by rdica** [reply]
Wow, on a side note the support link that app developer has points to the jamulus web site, so it looks like they dont provide any support for their app.

**Reply by dcorson-ticino-com** [reply]
I would suggest using a wired connection to any output devices as Bluetooth adds much latency, which you will not be happy with.

**Reply by Lightspacer** [reply]
> Is this the thing you installed:
> https://apps.apple.com/sn/app/jamulus2-0/id1609844773


yes, that is the app

**Reply by ann0see** [reply]
Yes. That's not official. Emlyn is working on a more friendly mobile app which is Jamulus compatible. You should NOT use bluetooth at all.

**Reply by ann0see** [reply]
> Volker Fischer is the main author

Yes, but the iOS version is beta-ish and not written by him. @ngocdh worked on the sound code, so maybe he knows a bit more about that. However, I could imagine that the sample rate doesn't match which is something you might not be able to solve over Bluetooth. So the suggestion is to use a cable (or adapter + wire)

**Reply by ngocdh** [reply]
Hi,
I wrote the iOS sound code. I'm sorry though I have no idea how to solve the bluetooth problem. I guess it has something to do with sample rate difference. 
Jamulus works fine via bluetooth on my Android phone (if you're just listenning).
FYI, the app you installed wasn't published by Jamulus official maintainers.

**Reply by dcorson-ticino-com** [reply]
At least under Win10 Jamulus runs fine with Bluetooth headphones.  So I doubt that this is a Jamulus problem.
But if you want to participate making music using Jamulus Bluetooth is deadly.  It adds way too much latency.
